bootcamp-java/topic2/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven

```

Comment:
ok, ya modifiqué el archivo gitignore

Date: Wed, 21 Jan 2015 11:56:30 -0800
From: notifications@github.com
To: java-bootcamp@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [java-bootcamp] Modificaciones solicitadas (#6)

In bootcamp-java/topic2/target/maven-archiver/pom.properties:

> @@ -0,0 +1,5 @@
> +#Generated by Maven

Ponele en el archivo .gitignore que ignore todo lo que esté en el directorio "target". De esta manera, no subís los archivos generados por Maven.


—
Reply to this email directly or view it on GitHub.

  
    
    
  
  
 		 	   		  =